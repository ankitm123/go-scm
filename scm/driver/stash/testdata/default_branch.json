{
  "id": "refs/heads/master",
  "displayId": "master",
  "type": "BRANCH",
  "latestCommit": "8d51122def5632836d1cb1026e879069e10a1e13",
  "latestChangeset": "8d51122def5632836d1cb1026e879069e10a1e13",
  "isDefault": true
}