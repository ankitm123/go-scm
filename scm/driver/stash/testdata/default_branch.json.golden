{
  "Name": "master",
  "Path": "refs/heads/master",
  "Sha": "8d51122def5632836d1cb1026e879069e10a1e13"
}